protocols/gossipsub/src/handler.rs

```diff
@@ -181,6 +181,13 @@ impl Handler {
             <Self as ConnectionHandler>::InboundOpenInfo,
         >,
     ) {
+        if self.inbound_substreams_created == MAX_SUBSTREAM_CREATION {
+            // Too many inbound substreams have been created, end the connection.
+            self.keep_alive = KeepAlive::No;
+            log::info!("Gossipsub error: The maximum number of inbound substreams created has been exceeded.");
```

Comment:
```suggestion
            log::info!("The maximum number of inbound substreams created has been exceeded.");
```

Nit: This is already part of the module that is inferred from where we use the macro.